python/ray/train/torch/torch_predictor.py

```diff
@@ -80,11 +102,10 @@ def tensorize(numpy_array, dtype):
             }
 
         with torch.no_grad():
-            self.model.eval()
             output = self.model(model_input)
 
         def untensorize(torch_tensor):
-            numpy_array = torch_tensor.cpu().detach().numpy()
+            numpy_array = torch_tensor.detach().cpu().numpy()
```

Comment:
this is minor nit improvement so we don't construct autograd edge on cpu then detach it. End result is the same.